runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1369,7 +1369,19 @@ scan_udata_memory_size_helper(J9JavaVM *javaVM, char **cursor, UDATA *value, con
 		return false;
 	}
 	
-	if(try_scan(cursor, "G") || try_scan(cursor, "g")) {
+	if(try_scan(cursor, "T") || try_scan(cursor, "t")) {
+		if (0 != value) {
```

Comment:
I expect you meant `(0 != *value)` (to dereference `value`).

Reply:
oops, my bad, let me fix that